### PR TITLE
Feature/additional datasets config

### PR DESCRIPTION
**Pull request by PresiDev**
Added 2 additional datasets and completed Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,34 +64,89 @@ data
...
   * `qdrant_storage` - storage for vector search engine (Qdrant), each data category has its own collection.
 
-### Local
+### Local - Manual
+
+Installation using the terminal window:
+* Install ***git***, ***docker*** packages.
+* `cd` to your target directory.
+* Clone [repository](https://github.com/stxnext/visual-similarity-search) (preferably use SSH cloning).
+* Download `data.zip` file from the [Google Drive](https://drive.google.com/file/d/1ru2FHcV8xBDqX9Q_zgAkJYDlYWlszmOD/view?usp=sharing) and unpack it to the repository so that the folder structure above is retained.
```
Comment (verlinen) [inline README.md]: Add some information why you need to download that and what it contains. Probably everyone will be asking themself the same question.

File: README.md [hunk 2]
```diff
@@ -64,34 +64,89 @@ data
...
+* Install Python version 3.10 and ***pip***, ***pipenv*** libraries.
+```
+sudo apt-get install python3.10 
+curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
+python3.10 -m pip install pipenv  
+```
+* Set up local environment and run shell:
+```
+python3.10 -m pipenv --python 3.10
```
Comment (verlinen) [inline README.md:108]: Do you really need pipenv here? I think that poetry has compatibility with pyenv.

File: README.md [hunk 3]
```diff
@@ -160,26 +222,103 @@ Application returns search results from the scope of images available only withi
...
+  * `weight_decay` - Weight decay for learning rate.
+  * `sampler_m` - Number of samples per class.
+  * `input_size` - Input size (width and height) used for resizing.
+
+To run the training module run the following command in the terminal (adjust based on above list of parameters).
+```
+python metrics/train.py --data_dir "data/metric_datasets/dataset_name" --meta "data/qdrant_storage/meta_dataset_name.csv" --name "metric_dataset_name"
+```
+
+After the training process is done, and final model is one more step to follow:
```
Comment (verlinen) [inline README.md]: After training follow steps:

File: interactive/__init__.py [hunk 4]
```diff
@@ -1,41 +1,97 @@
 from common.consts import PROJECT_PATH
 
-
 GRID_NROW_NUMBER = 3
 EXAMPLE_PATH = PROJECT_PATH / "interactive" / "examples"
 CATEGORY_DESCR = {
```
Comment (verlinen) [inline interactive/__init__.py]: I would suggest to move this big dict into separate file (could be consts etc.) and import it here instead of declaring

**Review by verlinen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PresiDev) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (PresiDev) [reply README.md:108]: Sometimes referenced poetry installation does not use the proper Python version defined in `.toml` files. Using shell takes care of that issue.

File: README.md (hunk 3 above)
Comment (PresiDev) [reply README.md]: Done

File: interactive/__init__.py (hunk 4 above)
Comment (PresiDev) [reply interactive/__init__.py]: Done
